website/api/security-model.md

```diff
@@ -77,13 +77,15 @@ The RISC-V Prover and the Recursion Prover both use STARK-based protocols, which
 
 ### How secure are the STARK Provers?
 
-We estimate that it would cost over 400 billion USD of computational resources to construct a fake proof for one of our STARK circuits.
+We estimate that it would take approximately 400,000 years to brute force a collision against our STARK protocol using GPUs in a parallel attack, using 1 million high-end GPUs.
 
-These estimates are based on the approach used in [this article by Justin Thaler].
-The napkin math here is as follows:
+- Currently, we look to the RTX 4090 as a high-end reference GPU.
+- RTX 4090 [benchmarks] provide a reference SHA2-256 hashrate at less than 25,000 million hashes per second.
+- We have seen several [examples] of large purchases of GPUs on the order of 350,000 so we conservatively suggest that a distributed attacker may be able to operate 1 million RTX 4090s in parallel.
```

Comment:
Hmm, slightly awkward to cite the 4090 and then link to a purchase of H100s. People often think of H100s as higher end than 4090s (certainly H100s are way more expensive), so this might look like we are unfairly weakening our case -- but actually the H100 benchmarks only just over 12,000 megahashes/second for SHA2-256 (https://gist.github.com/Chick3nman/e1417339accfbb0b040bcd0a0a9c6d54).

That said, 4090s are more recognizable to the not-immersed-in-GPU-stats-all-day audience, and I can't think of a concise way to explain this complexity, and this isn't really the core focus here. So my vote is: think about it for a couple minutes to see if you see any opportunities for revision I'm missing, and then probably just leave it as is.

Reply:
Possible alternative metric: Nvidia is expecting to put out "up to 2 million" H100s this year, up from around 550k last year. So we could perhaps cite using "every H100 ever made or projected to be made by the end of 2024" instead.

Reply:
Chatted with Tim over DMs about this point. 
We decided to leave it as is for now; we can always re-visit this later.